pages/search/index.tsx

```diff
@@ -20,33 +22,38 @@ export default function Search({ results }) {
   const { query } = useRouter()
   const searchParams = mapQueryToSearchParams(query)
 
+  const { itemListElement, totalResults } = results.results
+
+  const searchResultBibs = itemListElement.map((result: SearchResultsItem) => {
+    if (_isEmpty(result) || (result.result && _isEmpty(result.result))) {
+      return null
+    }
+    return new SearchResultsBib(result?.result)
```

Comment:
Does typescript give you a warning here if `?` is not used? I guess it can't tell if the previous logic check failed or not.

Reply:
There's no typescript warning since the if statement above catches the nullish value. Added it out of force of habit but will remove since that condition is unreachable.